toqito/state_props/abs_ppt_constraints.py

```diff
@@ -7,31 +7,31 @@
 def abs_ppt_constraints(
     eigs: np.ndarray | cp.Variable, p: int, max_constraints: int = 33_592, use_check: bool = False
 ) -> list[np.ndarray | cp.Expression]:
-    r"""Return the constraint matrices for the spectrum to be absolutely PPT :footcite:`Hildebrand_2007_AbsPPT`.
+    r"""Return the constraint matrices for the spectrum to be absolutely PPT :footcite:`Hildebrand_2007_AbsPPT` .
 
-    The returned matrices are constructed from the provided eigenvalues :code:`eigs`, and they must all be positive
+    The returned matrices are constructed from the provided eigenvalues :code:`eigs` , and they must all be positive
```

Comment:
We generally want the punctuation to follow directly the code/math rendered image (instead of moving it one space away as you are doing here).

Is there a reason this is done in this MR, that is, does it have anything directly to do with the fix for the issue?